crates/sncast/tests/e2e/declare.rs

```diff
@@ -95,68 +100,69 @@ async fn test_happy_case(class_hash: Felt, account_type: AccountType) {
     assert!(matches!(receipt, Declare(_)));
 }
 
-#[test_case(Some("100000000000000000"), None, None; "max_fee")]
-#[test_case(None, Some("100000"), None; "max_gas")]
-#[test_case(None, None, Some("100000000000000"); "max_gas_unit_price")]
-#[test_case(None, None, None; "none")]
-#[test_case(Some("10000000000000000000"), None, Some("100000000000000"); "max_fee_max_gas_unit_price"
-)]
-#[test_case(None, Some("100000"), Some("100000000000000"); "max_gas_max_gas_unit_price")]
-#[test_case(Some("100000000000000000"), Some("100000"), None; "max_fee_max_gas")]
-#[tokio::test]
-async fn test_happy_case_different_fees(
-    max_fee: Option<&str>,
-    max_gas: Option<&str>,
-    max_gas_unit_price: Option<&str>,
-) {
-    let contract_path = duplicate_contract_directory_with_salt(
-        CONTRACTS_DIR.to_string() + "/map",
-        "put",
-        &format!(
-            "{}{}{}",
-            max_fee.unwrap_or("0"),
-            max_gas.unwrap_or("1"),
-            max_gas_unit_price.unwrap_or("2")
-        ),
-    );
-    let tempdir = create_and_deploy_oz_account().await;
-    join_tempdirs(&contract_path, &tempdir);
-    let mut args = vec![
-        "--accounts-file",
-        "accounts.json",
-        "--account",
-        "my_account",
-        "--int-format",
-        "--json",
-        "declare",
-        "--url",
-        URL,
-        "--contract-name",
-        "Map",
-    ];
-
-    let options = [
-        ("--max-fee", max_fee),
-        ("--max-gas", max_gas),
-        ("--max-gas-unit-price", max_gas_unit_price),
-    ];
-
-    for &(key, value) in &options {
-        if let Some(val) = value {
-            args.append(&mut vec![key, val]);
-        }
-    }
-
-    let snapbox = runner(&args).current_dir(tempdir.path());
-    let output = snapbox.assert().success();
-
-    let output = output.get_output().stdout.clone();
-
-    let hash = get_transaction_hash(&output);
-    let receipt = get_transaction_receipt(hash).await;
-
-    assert!(matches!(receipt, Declare(_)));
-}
+// TODO(#3090)
+// #[test_case(Some("100000000000000000"), None, None; "max_fee")]
```

Comment:
I think that's a big one to actually ignore, I'd prefer we restore it in some form that before we release.

Reply:
Also let's uncomment this and ignore instead of making it commented out.

Reply:
I restored it with resource bounds case, the one with max fee is ignored (linked to #3100).